<?xml version="1.0"?>
<launch>

  <arg name="robot_ip" default="192.168.0.42"/>
  <arg name="topic_name" default="wrench"/>

  <node name="ft300data" pkg="ft300_wrench" type="ft300data.py" output="screen">
    <param name="robot_ip" type="string" value="$(arg robot_ip)"/>
    <param name="topic_name" type="string" value="$(arg topic_name)"/>
  </node>
</launch>

